### PR TITLE
SER-3063 Fix search-for-wiki-based tests

### PR DESCRIPTION
**Pull request by mkalinowski-wikia**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/search/crosswikisearch/CrossWikiSearchPageObject.java [hunk 1]
```diff
@@ -78,10 +78,12 @@ public CrossWikiSearchPageObject searchFor(String term) {
...
   }
 
-  public CrossWikiSearchPageObject navigateToWikiResults() {
-    wait.forElementVisible(otherCommunitiesLink);
-    scrollAndClick(otherCommunitiesLink);
-    return this;
+  public void searchForWiki(String searchUrl, String term) {
+    try {
+      getUrl(searchUrl + "/Special:Search?search=" + term);
+    } catch (TimeoutException e) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/search/crosswikisearch/CrossWikiSearchPageObject.java]: MAJOR Either log or rethrow this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by nlesniewski** [comment]
I am not convinced that it is solving the real issue.

Test 007 has never failed and it only searches for different phrase.

**Comment by mkalinowski-wikia** [comment]
There are few issues here. It solves x-wiki searching. It doesn't solve broken redirection, that is quite a different story.

**Pull request merged**